falcon/util/reader.py

```diff
@@ -354,22 +372,22 @@ def pipe_until(
                 raise DelimiterError('expected delimiter missing')
             self._buffer_pos += delimiter_len
 
-    def exhaust(self):
+    def exhaust(self) -> None:
         self.pipe()
 
-    def delimit(self, delimiter):
+    def delimit(self, delimiter: bytes) -> 'BufferedReader':
```

Comment:
add future annotation so we can avoid the string?

Reply:
What do you mean by "future annotation" @CaselIT?

Smth along the below lines?
```python
from __future__ import annotations

from copy import deepcopy


class Widget:
    ...

    def copy(self) -> Widget:
        return deepcopy(self)
```

Reply:
Yes

Reply:
Done.